src/commands/instance/spawn.ts

```diff
@@ -183,18 +184,18 @@ export default class InstanceSpawn extends Kommand {
       } else {
         cli.action.stop(
           chalk.red(
-            ` Something went wrong: docker-compose exited with ${docoCode}`
+            ` Something went wrong: docker compose exited with ${docoCode}`
           )
         );
         this.log(
           chalk.grey("If you want to investigate the problem, try running")
         );
         this.log(
           chalk.grey(
-            `  docker-compose -f ${docoFilename} -p stack-${portIndex} up\n`
+            `  docker compose -f ${docoFilename} -p stack-${portIndex} up\n`
           )
         );
-        throw new Error("docker-compose exited witn non-zero status");
+        throw new Error("docker compose exited witn non-zero status");
```

Comment:
I don't really know about capitalization here. If this text is printed directly to the user, having a lowercase first letter would be strange.

```suggestion
        throw new Error("docker compose exited with a non-zero status");
```